koku/masu/processor/tasks.py

```diff
@@ -668,9 +668,10 @@ def update_openshift_on_cloud(
         timeout = settings.WORKER_CACHE_TIMEOUT
         rate_limited = False
         fallback_queue = UPDATE_SUMMARY_TABLES_QUEUE
-        if is_customer_large(schema_name):
+        if is_rate_limit_customer_large:
```

Comment:
`is_rate_limit_customer_large` is a function...

Reply:
Dude its late OK... :smile: THANKYOU!!!

Reply:
In the previous function the value was stored in `is_large_customer_rate_limited`. Not confusing at all. 😉